source/hic_hal/nxp/lpc11u35/gpio.c

```diff
@@ -94,11 +112,19 @@ void gpio_init(void)
     PIN_CDC_LED_IOCON = PIN_CDC_LED_IOCON_INIT;
     LPC_GPIO->SET[PIN_CDC_LED_PORT] = PIN_CDC_LED;
     LPC_GPIO->DIR[PIN_CDC_LED_PORT] |= PIN_CDC_LED;
+#if defined(BOARD_LPC4088DM) || defined(BOARD_LPC4088QSB)
+    // Configure ISPCTRL as output and high
+    // We use two ISP pins to cover different hardware versions
+    LPC_IOCON->TMS_PIO0_12 |= 0x01;
+    LPC_IOCON->PIO1_15 &= ~0x07;
+    gpio_set_isp_pin(1);
+#else
```

Comment:
Do you mean to create a copy of the gpio.c (e.g. lpc4088_gpio.c in the same folder) and make the changes there instead or do you want weak functions? I'm all up for making the change but I need some guidelines on how to do it.